src/index.html

```diff
@@ -3,11 +3,353 @@
+              href="tel:+1234555-55-55" 
+              class="icon icon--phone header__call"
+            >
+            </a>
+
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            >
+            </a>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/MyBiKE-dark.svg"
+                alt="My bike"
+              >
+            </a>
+
+            <a href="/" class="icon icon--cross ">
+            </a>
+
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#compare-bikes" class="menu__link">Compare bikes</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a href="tel:+12345555-55-55" class="menu__call menu__call--phone">+1 234 5555-55-55</a>
+          <a href="tel:+12345555-55-55" class="menu__call menu__call--action">Book a test ride</a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="grid grid--tablet">
+            <div class="grid__item grid__item--tablet--1-3 grid__item--desktop--1-4">
+              <h2 class="page__section--title about__title">Move Free</h2>
+            </div>
+
+            <div class="grid__item grid__item--tablet--4-6 grid__item--desktop--7-12">
+              <div class="about__paragraph">
+                Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section compare" id="compare-bikes">
+        <div class="container">
+          <h2 class="page__section--title">Compare bikes</h2>
+
+          <div class="grid grid--tablet">
+            <div class="compare__item grid__item grid__item--tablet--2-5 grid__item--desktop--1-4 ">
+              <div class="bike">
+                <div class="bike__photo-container">
+                  <img
+                  class="bike__photo"
+                  src="./images/photos/image 16.jpg"
+                  alt="FirstBike">
```

Comment:
Need to fix code style everywhere
```suggestion
                  <img
                    class="bike__photo"
                    src="./images/photos/image 16.jpg"
                    alt="FirstBike"
                  >
```